### PR TITLE
feature: BB-166 change dmf location from aws s3

### PR DESCRIPTION
**Pull request by miniscruff**

### CODE REVIEW

File: lib/Config.js [hunk 1]
```diff
@@ -16,11 +16,11 @@ const locationTypeMatch = {
     'location-do-spaces-v1': 'aws_s3',
     'location-aws-s3-v1': 'aws_s3',
     'location-wasabi-v1': 'aws_s3',
-    'location-dmf-v1': 'aws_s3',
     'location-gcp-v1': 'gcp',
     'location-scality-ring-s3-v1': 'aws_s3',
     'location-scality-artesca-s3-v1': 'aws_s3',
     'location-ceph-radosgw-s3-v1': 'aws_s3',
+    'location-dmf-v1': 'location-dmf-v1',
```
Comment (rachedbenmustapha) [inline lib/Config.js]: The value `location-dmf-v1` is not consistent with the value `dmf` you added in configuration files generated by zenko-operator.

**Review by rachedbenmustapha — CHANGES_REQUESTED** [review]

File: lib/Config.js (hunk 1 above)
Comment (miniscruff) [reply lib/Config.js]: oh it matches the `type` value not the `locationType` value? So it should be just `dmf`?

Comment (rachedbenmustapha) [reply lib/Config.js]: Yes that's what this mapping does. Maybe there would be a way to deduplicate and remove this mapping (this pre-dates the `locationConfig.json` file) but would be out of scope.

**Review by rachedbenmustapha — APPROVED** [review]

**Review by nicolas2bert — APPROVED** [review]

**Pull request merged**